src/backends/mysql/session.cpp

```diff
@@ -148,6 +148,23 @@ bool valid_int(const string & s)
     return true;
 }
 
+bool valid_uint(const string & s)
+{
+    char *tail;
+    const char *cstr = s.c_str();
+    errno = 0;
+    unsigned long n = std::strtoul(cstr, &tail, 10);
+    if (errno != 0 || n > UINT_MAX)
+    {
```

Comment:
Minor comment, this could compress into

```
if (errno != 0 || n > UINT_MAX)
    return false;
```

Reply:
My implementation keeps consistency with `valid_int`.
https://github.com/SOCI/soci/blob/master/src/backends/mysql/session.cpp#L134-L149

Should I compress only `valid_uint`?

Reply:
Done. I updated only `valid_uint` I added.